src/Forms/NumericField.php

```diff
@@ -154,6 +154,9 @@ public function setValue($value, $data = null)
      */
     protected function cast($value)
     {
+        if (strlen($value) === 0) {
+            return false;
```

Comment:
Why is false being cast for a numeric field? I would expect `null` to be used here.